internal/diskutil/diskutil.go

```diff
@@ -166,6 +168,16 @@ func newSonoma(version semver.Version) (*diskutilSonoma, error) {
 	return du, nil
 }
 
+// newSequoia configures the DiskUtil for the specified Sequoia version.
+func newSequoia(version semver.Version) (*diskutilSonoma, error) {
```

Comment:
Are we planning on changing this to `*diskutilSequoia` at any point in the future?

Reply:
We will if we see that there's some need for Sequoia to have its own implementation for `disktuil`. However, it seems like it's OK for us to continue using the same setup for Sonoma for now.